devops-steps/src/main/java/io/fabric8/kubernetes/pipeline/devops/ApplyStepExecution.java

```diff
@@ -628,7 +636,11 @@ private String getRegistry() {
         if (Strings.isNullOrBlank(step.getRegistry())) {
             // lets try and find an external docker registry in the users home namespace pipeline config
             KubernetesClient client = getKubernetes();
-            ConfigMap cm = client.configMaps().inNamespace(this.buildConfigNamespace).withName(Constants.USERS_PIPELINE_CONFIGMAP_NAME).get();
+            String ns = this.buildConfigNamespace;
```

Comment:
In general I prefer to use readable variables instead of unpronunciable names. For example `ns` -> `namespace`/currentNamespace` or `cm` -> `configMap`. Sometimes it is really hard to find a name and the short name is used, but in general I prefer long pronunciable names.

Reply:
@lordofthejars , I agree with you, but ns, cm are used in the whole code base here and also Jenkins sync plugin too , but this change should not go with this PR we can have separate a PR for this change